### PR TITLE
docs(dashboard): initial commit for docs about the dashboard

### PR DESCRIPTION
**Pull request by mwasilew2**
# Description of the change

This PR starts a docs page for the Dashboard feature.

## Checklist

Please make sure that the proposed change checks all the boxes below before requesting a review:

- [ ] I have reviewed the [guidelines for contributing](https://github.com/spacelift-io/user-documentation/blob/main/CONTRIBUTING.md) to this repository.
- [ ] The preview looks fine.
- [ ] The tests pass.
- [ ] The commit history is clean and meaningful.
- [ ] The pull request is opened against the `main` branch.
- [ ] The pull request is no longer marked as a draft.
- [ ] You agree to license your contribution under the [MIT license](https://github.com/spacelift-io/user-documentation/blob/main/LICENSE) to Spacelift (not required for Spacelift employees).
- You have updated the navigation files correctly:
    - [ ] No new pages have been added, or;
    - [ ] Only _nav.yaml_ has been updated because the changes only apply to SaaS, or;
    - [ ] Only _nav.self-hosted.yaml_ has been updated because the changes only apply to Self-Hosted, or;
    - [ ] Both _nav.yaml_ and _nav.self-hosted.yaml_ have been updated.

If the proposed change is ready to be merged, please request a review from `@spacelift-io/solutions-engineering`. Someone will review and merge the pull request.

_Spacelift employees should request reviews from the relevant engineers and are allowed to merge pull requests after they got at least one approval._

Thank you for your contribution! 🙇

### CODE REVIEW

File: docs/product/dashboard.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Dashboard
+
+The dashboard is the first page you see when you log in to the app. It provides an overview of your account and the
+status of your infrastructure.
```
Comment (KNiepok) [inline docs/product/dashboard.md:4]:
```suggestion
The dashboard is the first page you see when logging into the app. It provides an overview of your account and the
status of your infrastructure.
```

File: docs/product/dashboard.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+The Dashboard comes with two predefined views: `Overview` and `Metrics`. You can switch between them by clicking on the
+tabs at the top of the page.
+
+You can also create your own views to customize the dashboard to your needs. To create a new view, click on the `+`
+button next to the existing tabs.
+
+![](../assets/screenshots/dashboard_adding_views.png)
+
+"Manage views" dialog will appear. Here you can customize the layout settings and choose the widgets you want to display.
```
Comment (KNiepok) [inline docs/product/dashboard.md:16]:
```suggestion
The "Manage views" dialog will appear. Here you can customize the layout settings and choose the widgets you want to display.
```

File: docs/product/dashboard.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+![](../assets/screenshots/dashboard_manage_views.png)
+
+## Changing landing page
```
Comment (KNiepok) [inline docs/product/dashboard.md:20]:
```suggestion
## Changing the landing page
```

**Review by KNiepok — DISMISSED** [review]
minor review notes with some Grammarly help

**Comment by mwasilew2** [comment]
fixing of a broken link is unrelated to this change

**Review by KNiepok — APPROVED** [review]

**Pull request merged**